superbench/analyzer/data_diagnosis.py

```diff
@@ -129,8 +128,7 @@ def _parse_rules_and_baseline(self, rules, baseline):
                 self.__get_metrics_and_baseline(rule, benchmark_rules, baseline)
             self._enable_metrics = sorted(list(self._enable_metrics))
         except Exception as e:
-            logger.error('DataDiagnosis: get criteria failed - {}'.format(str(e)))
-            return False
+            logger.log_and_raise(exception=Exception, msg='DataDiagnosis: get criteria failed - {}'.format(str(e)))
```

Comment:
why do you need `str()` in `format()`?

Reply:
not necessary, but to keep consistency with other files...